core/src/main/clojure/xtdb/compactor.clj

```diff
@@ -71,6 +72,8 @@
 
           (.end trie-wtr))))))
 
+(def ^:private constant-all-page-bitmap (RoaringBitmap/flip (RoaringBitmap.) 0 (bit-shift-left 1 16)))
```

Comment:
My understanding of how many pages there ever can be in a trie. Not sure if that holds.